aten/src/ATen/native/quantized/library.cpp

```diff
@@ -152,6 +152,7 @@ TORCH_LIBRARY(quantized, m) {
   m.def(TORCH_SELECTIVE_SCHEMA("quantized::linear_relu_dynamic(Tensor X, __torch__.torch.classes.quantized.LinearPackedParamsBase W_prepack, bool reduce_range=False) -> Tensor Y"));
   m.def(TORCH_SELECTIVE_SCHEMA("quantized::linear_dynamic_fp16(Tensor X, __torch__.torch.classes.quantized.LinearPackedParamsBase W_prepack) -> Tensor Y"));
   m.def(TORCH_SELECTIVE_SCHEMA("quantized::linear_relu_dynamic_fp16(Tensor X, __torch__.torch.classes.quantized.LinearPackedParamsBase W_prepack) -> Tensor Y"));
+  m.def(TORCH_SELECTIVE_SCHEMA("quantized::linear_leaky_relu(Tensor X, __torch__.torch.classes.quantized.LinearPackedParamsBase W_prepack, float Y_scale_i, int Y_zero_point_i, float negative_slope) -> Tensor Y"));
```

Comment:
nit: what does `i` in `Y_scale_i` mean?

Reply:
> nit: what does `i` in `Y_scale_i` mean?

Good question. I don't know. I just copied the arg name from above to look aligned https://github.com/pytorch/pytorch/blob/55b10e6b1d52aeae536129d152b1d057f159bfc0/aten/src/ATen/native/quantized/library.cpp#L150
Shall I keep it or remove it?